### PR TITLE
Figure readme updates

### LINKED ISSUE #20: Update readme
Readme could be slightly more descriptive 😉

### PR DESCRIPTION
**Pull request by danich1**
Closes #20.

This PR updates the readme and fixes a few figures for the manuscript. No in depth review needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
-# annorxiver
-Annotating Rxiv preprints in an automated fashion
+# Annorxiver
+
+## Citation
+- [Manuscript in progress](greenelab.github.io/annorxiver_manuscript)
```
Comment (ajlee21) [inline README.md]:
```suggestion
- [Manuscript in progress](https:/greenelab.github.io/annorxiver_manuscript)
```

Comment (ajlee21) [reply README.md]: Link didn't work, but i think this should fix it. Tested it already

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+- doi and full citation coming soon!!
+
+## About
+This repository contains code for the annorxvier project.
+This project focuses on analyzing the linguistic style and content of bioRxiv preprints and understanding how these features change when preprints undergo the publication process.
```
Comment (ajlee21) [inline README.md]:
```suggestion
This project analyzes the linguistic style and content of bioRxiv preprints and aims to understand how these features change when preprints undergo the publication process.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+We are currently working on a manuscript for the work in this repository.
+Feel free to check it out! (link above).
```
Comment (ajlee21) [inline README.md:11]: Not sure if you will keep this sentence in the future, but if you do, maybe just add the link here `(link above)` >> `(link [here](https:/greenelab.github.io/annorxiver_manuscript)`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,51 @@
...
+We also created a [web app](greenelab.github.io/preprint-similarity-search) allows anyone to input a bioRxiv or medRxiv preprint and receive a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.
```
Comment (ajlee21) [inline README.md]:
```suggestion
We also created a [web app](greenelab.github.io/preprint-similarity-search) that takes a bioRxiv or medRxiv preprint doi as input and returns a set of the most linguistically similar journals and articles to serve as potential publication venues for their work.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,51 @@
...
+
+## Directory Structure
+| Folder/file | Description |
+| --- | --- | 
+| [annorxiver_modules](annorxiver_modules) | This folder contains universal functions that other notebooks in this repository will use |
```
Comment (ajlee21) [inline README.md]:
```suggestion
| [annorxiver_modules](annorxiver_modules) | This folder contains supporting functions that other notebooks in this repository will use |
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,51 @@
...
+| [biorxiv](biorxiv) | This folder contains all experiments that are related to biorxiv preprints. | 
+| [credentials](credentials) | Some of the code in this repository needs credentials to run. Any users that need to run those notebooks use check this folder first. |
```
Comment (ajlee21) [inline README.md]:
```suggestion
| [credentials](credentials) | Some of the code in this repository need credentials to run. Any user that needs to run those notebooks should check this folder first. |
```

Comment (ajlee21) [reply README.md]: Do all notebooks need a user to fill out credentials? I assume this just pertains to the download of the data. If so, maybe you could add a separate section for how to run the experiments.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,51 @@
...
+| [figure_generation](figure_generation) | This folder contains code to generate figures for the manuscript in progress. |
+| [nytac/corpora_stats](nytac/corpora_stats) | This folder contains results when parsing the New York Times annotated corpus (NYTAC) from the Linguistic Data Consortium (LDC). |
+| [pmc](pmc) |  This folder contains all experiments that are related to articles in Pubmed Central Open Access corpus (PMC). | 
+| environment.yml | This file contains the necessary packages this repository uses.  |
```
Comment (ajlee21) [inline README.md]: Do you want to add links to these files too?

File: README.md [hunk 8]
```diff
@@ -1,2 +1,51 @@
...
+| setup.py | This file sets up the annorxiver modules to be used as a regular python package. |
+
+
+## Set up Environment
+
+Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://www.continuum.io/downloads)
```
Comment (ajlee21) [inline README.md]: The continuum link seems to be broken

File: biorxiv/README.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Directory Structure
+
+| Folder/file | Description |
+| --- | --- | 
+| [article_distances](article_distances) | This folder contains notebooks that aimed to find missing links within bioRxiv. |
+| [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aimed to compare bioRxiv with PMC and NYTAC. | 
+| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtains metadata from the bioRxiv respository. |
```
Comment (ajlee21) [inline biorxiv/README.md]:
```suggestion
| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtain metadata from the bioRxiv respository. |
```

File: biorxiv/README.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+| [journal_tracker](journal_tracker) | This folder contains code map preprints to their published counterparts. |
```
Comment (ajlee21) [inline biorxiv/README.md]:
```suggestion
| [journal_tracker](journal_tracker) | This folder contains code that maps preprints to their published counterparts. |
```

File: biorxiv/README.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+# Directory Structure
+
+| Folder/file | Description |
```
Comment (ajlee21) [inline biorxiv/README.md:3]: Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere

File: pmc/README.md [hunk 12]
```diff
@@ -0,0 +1,6 @@
+# Directory Structure
+
+| Folder/file | Description |
+| --- | --- | 
+| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtains metadata from the bioRxiv respository. |
```
Comment (ajlee21) [inline pmc/README.md]:
```suggestion
| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtain metadata from the bioRxiv respository. |
```

File: pmc/README.md [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+| [journal_recommendation](journal_recommendation) | This folder contains code aims to return a set of journals that publish linguistically similar articles to a biorxiv or medrxiv preprint. |
```
Comment (ajlee21) [inline pmc/README.md]:
```suggestion
| [journal_recommendation](journal_recommendation) | This folder contains code that aims to return a set of journals that publish linguistically similar articles to a biorxiv or medrxiv preprint. |
```

File: pmc/README.md [hunk 14]
```diff
@@ -0,0 +1,6 @@
+# Directory Structure
+
+| Folder/file | Description |
```
Comment (ajlee21) [inline pmc/README.md:3]: Similar comment as before, if there is an order to running the notebooks that might be helpful to add here

**Review by ajlee21 — APPROVED** [review]
Really nice clean readme - very well organized. Love the table, its very helpful!

Just have a bunch of minor text edits (feel free to take/leave what you like)

A couple of notes on the notebooks:
For fig 4B, I wonder if its worth adding a note in your caption to say that the violin plots are more uniform a x>7 because you have fewer samples. I don't think it changes your takeaway for this figure. Just a thought in case a reader asks.

Minor, looks like your time delay plots were updated in the `manuscript_figure_generation.ipynb` but not in the `publication_delay_experiment.ipynb` -- i.e. they are boxplots instead of violin plots

File: README.md [hunk 15]
```diff
@@ -1,2 +1,51 @@
-# annorxiver
-Annotating Rxiv preprints in an automated fashion
+# Annorxiver
+
+## Citation
+- [Manuscript in progress](greenelab.github.io/annorxiver_manuscript)
+- doi and full citation coming soon!!
+
+## About
+This repository contains code for the annorxvier project.
```
Comment (ben-heil) [inline README.md]:
```suggestion
This repository contains code for the annorxiver project.
```

File: README.md (hunk 8 above)
Comment (ben-heil) [inline README.md]:
```suggestion
Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://docs.conda.io/en/latest/miniconda.html)
```

File: README.md [hunk 16]
```diff
@@ -1,2 +1,51 @@
...
+| [pmc](pmc) |  This folder contains all experiments that are related to articles in Pubmed Central Open Access corpus (PMC). | 
+| environment.yml | This file contains the necessary packages this repository uses.  |
+| setup.py | This file sets up the annorxiver modules to be used as a regular python package. |
+
+
+## Set up Environment
+
+Annorxiver uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://www.continuum.io/downloads)
+  
+Once everything has been installed, type following command in the terminal:
```
Comment (ben-heil) [inline README.md:31]: May want to say "Navigate to the annorxiver directory and type the following command in the terminal"

File: biorxiv/README.md [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+| --- | --- | 
+| [article_distances](article_distances) | This folder contains notebooks that aimed to find missing links within bioRxiv. |
```
Comment (ben-heil) [inline biorxiv/README.md]:
```suggestion
| [article_distances](article_distances) | This folder contains notebooks that aim to find missing links within bioRxiv. |
```

File: biorxiv/README.md [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+| [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aimed to compare bioRxiv with PMC and NYTAC. |
```
Comment (ben-heil) [inline biorxiv/README.md]:
```suggestion
| [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aim to compare bioRxiv with PMC and NYTAC. | 
```

File: biorxiv/README.md [hunk 19]
```diff
@@ -0,0 +1,13 @@
+# Directory Structure
+
+| Folder/file | Description |
+| --- | --- | 
+| [article_distances](article_distances) | This folder contains notebooks that aimed to find missing links within bioRxiv. |
+| [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aimed to compare bioRxiv with PMC and NYTAC. | 
+| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtains metadata from the bioRxiv respository. |
+| [journal_tracker](journal_tracker) | This folder contains code map preprints to their published counterparts. |
+| [missing_sections_experiment](missing_sections_experiment) | This folder contains notebooks that aimed to find if a preprint is missing a section. |
```
Comment (ben-heil) [inline biorxiv/README.md]:
```suggestion
| [missing_sections_experiment](missing_sections_experiment) | This folder contains notebooks that aim to find if a preprint is missing a section. |
```

**Review by ben-heil — APPROVED** [review]
Looks good! Added some suggested edits that make tenses consistent across files etc

File: biorxiv/README.md (hunk 11 above)
Comment (danich1) [reply biorxiv/README.md:3]: >Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere 

That is correct. There isn't a specified order to run notebooks.

**Pull request merged**